### PR TITLE
feat(replays): add FAQ section to replay onboarding

### LINKED ISSUE #61760: [Replay Index] FAQ section
SCR-20231214-hymg

### PR DESCRIPTION
**Pull request by michellewzhang**
- One panel is visible at all times (accordion is not collapsible)
- The topmost panel is open by default

https://github.com/getsentry/sentry/assets/56095982/b2b953f6-6e1a-4db1-981b-14bfa2c096d3

Also fixed a small bug with the accordion where clicking the panel itself doesn't collapse it.

Closes https://github.com/getsentry/sentry/issues/61760

### CODE REVIEW

**Review by c298lee — APPROVED** [review]

File: static/app/components/accordion/accordion.tsx [hunk 1]
```diff
@@ -89,7 +89,11 @@ function AccordionItem({
     <StyledLineItem>
       <ButtonLeftListItemContainer>
         {button}
-        <StyledPanel onClick={() => setExpandedIndex(index)}>{children}</StyledPanel>
+        <StyledPanel
+          onClick={() => (isExpanded ? setExpandedIndex(-1) : setExpandedIndex(index))}
```
Comment (billyvg) [inline static/app/components/accordion/accordion.tsx:93]: style nit:

```suggestion
          onClick={() => setExpandedIndex(isExpanded ? -1 : index)}
```

File: static/app/views/replays/list/replayOnboardingPanel.tsx [hunk 2]
```diff
@@ -141,6 +144,81 @@ export function SetupReplaysCTA({
...
+      content: () => (
+        <AnswerContent>
+          <div>
+            {t(
+              'Session Replay supports all browser-based applications. This includes static websites, single-page-aplications, and also server-side-rendered. The only prerequisite is that your application uses our Sentry JavaScript SDK (7.2.0 or greater) either with NPM/Yarn or our JS Loader script.'
+            )}
+          </div>
+          <div>
+            {tct(
+              'To learn about which SDKs we support, please visit [link:this page] from our docs.',
```
Comment (billyvg) [inline static/app/views/replays/list/replayOnboardingPanel.tsx:162]: For these links, should we shorten it to something like:

`..., please visit [our docs]`

File: static/app/views/replays/list/replayOnboardingPanel.tsx [hunk 3]
```diff
@@ -141,6 +144,81 @@ export function SetupReplaysCTA({
...
+  const FAQ = [
+    {
+      header: () => (
+        <QuestionContent>{t('Can I use Session Replay with my app?')}</QuestionContent>
+      ),
+      content: () => (
+        <AnswerContent>
+          <div>
+            {t(
+              'Session Replay supports all browser-based applications. This includes static websites, single-page-aplications, and also server-side-rendered. The only prerequisite is that your application uses our Sentry JavaScript SDK (7.2.0 or greater) either with NPM/Yarn or our JS Loader script.'
```
Comment (billyvg) [inline static/app/views/replays/list/replayOnboardingPanel.tsx:157]:
```suggestion
              'Session Replay supports all browser-based applications. This includes static websites, single-page applications, and also server-side rendered applications. The only prerequisite is that your application uses Sentry JavaScript SDK (7.2.0 or greater) either with NPM/Yarn or our JS Loader script.'
```

File: static/app/views/replays/list/replayOnboardingPanel.tsx [hunk 4]
```diff
@@ -141,6 +144,81 @@ export function SetupReplaysCTA({
...
+    },
+    {
+      header: () => (
+        <QuestionContent>{t('What’s the performance overhead?')}</QuestionContent>
+      ),
+      content: () => (
+        <AnswerContent>
+          <div>
+            {t(
+              'Session Replay adds a small amount of performance overhead to your web application. The performance overhead basically scales linearly with the amount of DOM complexity your application has. The more DOM state changes that occur in the application lifecycle, the more events that are captured, transmitted, etc.'
```
Comment (billyvg) [inline static/app/views/replays/list/replayOnboardingPanel.tsx:181]:
```suggestion
              'Session Replay adds a small amount of performance overhead to your web application. The performance overhead generally scales linearly with the amount of DOM complexity your application has. The more DOM state changes that occur in the application lifecycle, the more events that are captured, transmitted, etc.'
```

**Pull request merged**